src/stratigraphy/data_extractor/data_extractor.py

```diff
@@ -204,21 +203,38 @@ def get_lines_near_key(self, lines, key_line: TextLine) -> list[TextLine]:
 
         return feature_lines_sorted
 
-    def get_lines_near_rect(self, lines, rect: fitz.Rect) -> list[TextLine]:
-        """Find the lines of the text that are close to a given rectangle.
+    def get_axis_aligned_lines(self, lines: list[TextLine], rect: fitz.Rect) -> list[TextLine]:
```

Comment:
I would suggest writing a few unit tests for this method. As this method is part of the abstract class DataExtractor, we will have to pick a particular implementation for the unit tests. For coordinate extraction, we already have a few similar methods that are tested in `tests/test_coordinate_extraction.py`, so you can look there for inspiration. The actual tests for `get_axis_aligned_lines` could be added there for `CoordinateExtractor`, but you could also write them for `ElevationExtractor` or `GroundwaterExtractor` (one is enough for now, I think).